src/components/graph-view.js

```diff
@@ -729,6 +758,10 @@ class GraphView extends Component {
   renderNodes = (entities, nodes) => {
     var self = this;
     const nodeKey = this.props.nodeKey;
+    const parent = this.viewWrapper;
+    const viewTransform = this.state.viewTransform;
+    const overlap = (300 / viewTransform.k);
```

Comment:
300 is a magic number as a first step. Ideally there would be a way to get the node's width and height but it hasn't rendered yet, so we only know where the center will be, not that even if the center is off the canvas the node itself might be rendering on the canvas.